src/lib/classes/Quaff.svelte.ts

```diff
@@ -32,6 +36,10 @@ class Quaff {
       set: (newVal: boolean) => this.setDarkMode(newVal),
     };
   }
+
+  public get router() {
```

Comment:
It'll do for now but this function is useless as is. Svelte 5 will most likely introduce a function to handle routing (like today's page store)